app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java

```diff
@@ -41,6 +41,8 @@
 
     private ContentSettingsManager manager;
 
+    private String importExportDataPathKey;
```

Comment:
Any reason for it to be an member variable instead of just `getString(R.string.import_export_data_path)` putting whereever its used. I don't see it as a good reason, since it can be easily fetched from that

Reply:
I have followed older codes that puts `R.string.download_thumbnail_key` and `getString(R.string.youtube_restricted_mode_enabled)` in member variables and I think the code is cleaner with don't repeating `getString(R.string.import_export_data_path)`.

If you disagree with me, I'll fix it when I find free time.